src/Nirum/Targets/Python.hs

```diff
@@ -200,6 +207,9 @@ runCodeGen :: CodeGen a
            -> (Either CompileError' a, CodeGenContext)
 runCodeGen = C.runCodeGen
 
+renderCompileText :: BI.Markup -> T.Text
+renderCompileText = toStrict . renderMarkup
```

Comment:
These `toStrict $ renderMarkup ...` codes have remained by intention; they need to be merely reduced to `renderMarkup` without `toStrict` in the near future by replacing `type CompileResult Python = Code` with `type CompileResult Python = BI.Markup`.